src/libraries/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -823,6 +823,9 @@ public static IEnumerable<object[]> Groups_Basic_TestData()
             // Empty Match
             yield return new object[] { null, @"([a*]*)+?$", "ab", RegexOptions.None, new string[] { "", "" } };
             yield return new object[] { null, @"(a*)+?$", "b", RegexOptions.None, new string[] { "", "" } };
+
+            // A unit test from aspnetcore
```

Comment:
Can we instead say what's special about this particular pattern? i.e. this obviously covers something that literally none of our other tests do... what is it?